### PR TITLE
Bump sentry sdk to 4.0.0-alpha.1

### PR DESCRIPTION
**Pull request by marandaneto**
## :loudspeaker: Type of change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [X] Refactoring

## :scroll: Description
Bump sentry sdk to 4.0.0-alpha.1

## :bulb: Motivation and Context
preparing to release the first alpha of the major v4

## :pencil: Checklist

- [X] I reviewed submitted code
- [ ] I added tests to verify changes
- [X] All tests passing
- [ ] No breaking changes

### CODE REVIEW

File: flutter/example/lib/main.dart [hunk 1]
```diff
@@ -11,6 +11,7 @@ import 'package:universal_platform/universal_platform.dart';
 const String _release =
     String.fromEnvironment('SENTRY_RELEASE', defaultValue: 'unknown');
 
+// Change the DSN
```
Comment (bruno-garcia) [inline flutter/example/lib/main.dart]:
```suggestion
// ATTENTION: Change the DSN below with your own to see the events in Sentry. Get one at sentry.io
```

**Review by bruno-garcia — APPROVED** [review]
excited to see this!!

Maybe a stronger note on the DSN and we're good to go

File: README.md [hunk 2]
```diff
@@ -83,43 +78,39 @@ main() async {
...
-        sentry.captureException(
-          exception: error,
-          stackTrace: stackTrace,
-        );
-        print('Error sent to sentry.io: $error');
-      } catch (e) {
-        print('Sending report to sentry.io failed: $e');
-        print('Original error: $error');
-      }
+      Sentry.captureException(
```
Comment (marandaneto) [inline README.md]:
```suggestion
      await Sentry.captureException(
```

File: README.md [hunk 3]
```diff
@@ -52,29 +52,24 @@ Add `sentry` dependency to your `pubspec.yaml`:
 
 ```yaml
 dependencies:
-  sentry: ">=3.0.1 <4.0.0"
+  sentry: ^4.0.0
 ```
 
-In your Dart code, import `package:sentry/sentry.dart` and create a `SentryClient` using the DSN issued by Sentry.io:
+In your Dart code, import `package:sentry/sentry.dart` and `Sentry.init(...)` using the DSN issued by Sentry.io:
```
Comment (marandaneto) [inline README.md]:
```suggestion
In your Dart code, import `package:sentry/sentry.dart` and initialize the Sentry SDK using the DSN issued by Sentry.io:
```

File: README.md [hunk 4]
```diff
@@ -52,26 +52,29 @@ Add `sentry` dependency to your `pubspec.yaml`:
...
 ```
 
-In your Dart code, import `package:sentry/sentry.dart` and create a `SentryClient` using the DSN issued by Sentry.io:
+In your Dart code, import `package:sentry/sentry.dart` and initialize the Sentry SDK using the DSN issued by Sentry.io:
 
 ```dart
 import 'package:sentry/sentry.dart';
 
-final SentryClient sentry = new SentryClient(dsn: YOUR_DSN);
+Sentry.init((options) => options.dsn = '___PUBLIC_DSN___');
```
Comment (bruno-garcia) [inline README.md]:
```suggestion
Sentry.init((options) => options.dsn = 'https://example@sentry.io/add-your-dsn-here');
```

**Pull request merged**